File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task69_PERPLEXITY_llama-3-sonar-large-32k-chat.cpp
#include <iostream>
#include <fstream>

int main(int argc, char *argv[]) {
    if (argc != 2) {
        std::cout << "Usage: " << argv[0] << " <file_path>" << std::endl;
        return 1;
    }
    if (std::remove(argv[1]) == 0) {
        std::cout << "File deleted successfully" << std::endl;
    } else {
        std::cout << "Failed to delete the file" << std::endl;
    }
    return 0;
}

File: dataset/output-bk/PERPLEXITY_llama-3-sonar-large-32k-chat/Task69_PERPLEXITY_llama-3-sonar-large-32k-chat.java
import java.io.File;
import java.io.IOException;

public class DeleteFile {
    public static void main(String[] args) {
        if (args.length != 1) {
            System.out.println("Usage: java DeleteFile <file_path>");
            return;
        }
        File file = new File(args[0]);
        if (file.delete()) {
            System.out.println("File deleted successfully");
        } else {
            System.out.println("Failed to delete the file");
        }
    }
}